### ISSUE TITLE
Konfiguration laden/speichern button

### ISSUE THREAD

**Issue by bahnuhr** [ioBroker/ioBroker.modbus#270]
o.g. button fehlt in der config.

Bei anderen Adaptern ist dies standard.

mfg
D.Müller

**Comment by Apollon77** [comment]
Das die Liste der Register import und export hat ist bekannt?

**Comment by bahnuhr** [comment]
> Das die Liste der Register import und export hat ist bekannt?

Nein, das ist nicht bekannt.
Und in der config der Instanz auch nicht vorhanden. Danach hatte ich schon einmal vergeblich gesucht.

Bitte einen Hinweis wo dies zu finden ist.
Danke.

**Comment by Apollon77** [comment]
![Bildschirm­foto 2022-11-24 um 15 38 14](https://user-images.githubusercontent.com/11976694/203810052-869340c3-dd3a-4a17-9b56-5700ebf5efc0.png)
![Bildschirm­foto 2022-11-24 um 15 38 27](https://user-images.githubusercontent.com/11976694/203810095-1c6c4622-6a9b-4d44-8bca-a38b92626f04.png)

**Comment by bahnuhr** [comment]
Warum wird ein speichern button nicht eingefügt ?
War das nicht irgendwann mal pflicht als Aufgabe für den Adapterentwickler.
ab admin 5 ?
oder etwas anderes, ich weiß nicht mehr.

![image](https://user-images.githubusercontent.com/25694404/203811380-5cde70f4-12ca-43a7-83df-cda15065f580.png)

**Comment by bahnuhr** [comment]
Und wenn es nicht gewünscht ist, dann sollte man es auch sagen.
Ein Ablaufen der Issues ohne Antwort ist halt schlecht.

**Comment by Apollon77** [comment]
Das der nicht da ist (und  ist was von Admin und nicht vom Entwickler) liegt daran das der Adapter hier eine andere Art von Oberfläche hat. Also ja der fehlt immer noch ... und bei einigen Adaptern macht der keinen Sinn weil die Daten wo ganz anders liegen. Aber wenn es darum geht die Daten der Register zu sichern ist der TSV Export/Import von oben "the way to go".
Der "andere Button" wäre nur ein" Auch die restlichen Settings" sichern.

**Comment by Apollon77** [comment]
Es hat keiner was gesagt weil keienr Zeit hatte sich das Thema anzusehen. Ich gebe auch nur Tipps was Alternativ gemacht werden kann. Ich entscheide nicht ob das issue gemacht wird oder nicht

**Comment by bahnuhr** [comment]
Ist ja gut, ich wollte auch gar nichts mehr sagen. Meine issues laufen i.d.R. per Zeitablauf ab. Und es hat sich nichts geändert.

**Comment by Apollon77** [comment]
Dann reiocht einkurzes "noch aktuell" alle 3 Monate ...

**Comment by bahnuhr** [comment]
Nein, das wird auch nicht beantwortet. Außerdem weiß ich wirklich nicht, wie ein aktiver Benutzer dies überblicken soll; dann müsste ich ja eine Liste separat führen, was ich wo, wann und warum als issue gesetzt habe. In der Praxis ist dies nicht umstetzbar.
Wenn sich nichts ändert ist dies natürlich aktuell.
Aber diese Diskussion hatten wir ja schon.

**Comment by Apollon77** [comment]
(kriegst doch emails wenn stalebot die issues kommentiert) ;-) (und ja hatten wir)
